i2s/nau8825-tplg.xml

```diff
@@ -71,6 +71,7 @@
             <UUID>9BA00C83-CA12-4A83-943C-1FA2E82F9DDA</UUID>
             <CprOutAudioFormatId>1</CprOutAudioFormatId>
             <CprFeatureMask>0</CprFeatureMask>
+            <CprVirtualIndex>2</CprVirtualIndex>
```

Comment:
Can't we define this in machine board definitions and somehow assign it dynamically, similarly to SSP?

Reply:
That would probably require usage of DMI i.e.: tie vindex value to specific device.

Reply:
We both agreed to the following approach:
- upstream the topology as is
- add a static DMI-based table within the kernel to modify virtual index for selected devices automatically
- update the topology accordingly once the above lands on the list

Reply:
ACK